services/cinema_hall.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
You could indicate that you return a queryset of `CinemaHall` instances in the type annotation.